services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_utils.py

```diff
@@ -89,13 +89,15 @@ def parse_containers_inspect(
     return list(results)
 
 
-def all_containers_running(containers_inspect: List[DockerContainerInspect]) -> bool:
+def are_all_service_containers_running(
+    containers_inspect: List[DockerContainerInspect],
+) -> bool:
     return len(containers_inspect) > 0 and all(
         (x.status == DockerStatus.RUNNING for x in containers_inspect)
     )
 
 
-async def attempt_user_create_services_removal_and_data_saving(
+async def attempt_user_created_services_removal_and_data_saving(
```

Comment:
for the naming above, then you could use a convention like:
- user_services = the car or the service as in the GUI
- sidecar, proxy = obvious
- pod = sidecar,proxy+user_services

Reply:
> * user_services = the car or the service as in the GUI
> * sidecar, proxy = obvious

I like these a lot

> * pod = sidecar,proxy+user_services

I like it less but it does the job

Reply:
Let's go with this, I've put it int he `__init__.py` of the `dynamic_sidecar` module of `director-v2`